mne/time_frequency/tests/test_csd.py

```diff
@@ -110,47 +112,47 @@ def test_compute_epochs_csd():
     assert_array_equal(csd_fsum.data, csd_sum)
 
 
-def test_compute_epochs_csd_on_artificial_data():
+def test_csd_epochs_on_artificial_data():
     """Test computing CSD on artificial data
     """
-    epochs, epochs_sin = _get_data()
-    sfreq = epochs_sin.info['sfreq']
+    epochs = _get_data(mode='sin')
+    sfreq = epochs.info['sfreq']
 
     # Computing signal power in the time domain
-    signal_power = sum_squared(epochs_sin._data)
-    signal_power_per_sample = signal_power / len(epochs_sin.times)
+    signal_power = sum_squared(epochs._data)
+    signal_power_per_sample = signal_power / len(epochs.times)
 
     # Computing signal power in the frequency domain
-    data_csd_fourier = compute_epochs_csd(epochs_sin, mode='fourier')
-    data_csd_mt = compute_epochs_csd(epochs_sin, mode='multitaper')
+    data_csd_fourier = csd_epochs(epochs, mode='fourier')
+    data_csd_mt = csd_epochs(epochs, mode='multitaper')
     fourier_power = np.abs(data_csd_fourier.data[0, 0]) * sfreq
     mt_power = np.abs(data_csd_mt.data[0, 0]) * sfreq
     assert_true(abs(fourier_power - signal_power) <= 0.5)
     assert_true(abs(mt_power - signal_power) <= 1)
 
     # Power per sample should not depend on time window length
-    for tmax in [0.2, 0.4, 0.6, 0.8]:
+    for tmax in [0.2, 0.8]:
         for add_n_fft in [30, 0, 30]:
```

Comment:
I see you use this elsewhere -- why repeat the 30 condition? Does something accumulate / change?